qutebrowser/browser/webengine/darkmode.py

```diff
@@ -236,6 +237,19 @@ def copy_add_setting(self, setting: _Setting) -> '_Definition':
         new._settings = self._settings + (setting,)  # pylint: disable=protected-access
         return new
 
+    def copy_replace_setting(self, old: _Setting, new: _Setting) -> '_Definition':
```

Comment:
What do you think about making this take the qutebrowser option as `str` and the new `chromium_key` as `str`, and then mutating the existing `_Setting` object? Seems like it might make things a little more straightforward, and I don't really see a case where we'd want to replace an entire `_Setting` object.

Reply:
I updated the signature. Since the global `DEFINITIONS` instance just holds references to `_Definition`s and their `_Setting`s, we still need to do a `copy.deepcopy()` here.
Otherwise mutating a qt6 `_Setting` object is also reflected in the qt5 `_Definition`, which breaks the tests.